app/src/main/java/fr/groggy/racecontrol/tv/core/season/SeasonService.kt

```diff
@@ -99,7 +100,7 @@ class SeasonService @Inject constructor(
 
     private fun thumbnail(session: F1TvSession): Flow<Image> {
         return flowOf(
-            Image(Uri.parse(session.pictureUrl))
+            Image(Uri.parse(session.largePictureUrl)) // We use the large picture url since a better quality image is preferred on a large screen
```

Comment:
Revert this change.

The thing is that doing this we are caching more stuff, for some devices as the Google Chrome cast with google tv, we can easily fill up the storage, and also the card isn't 1080p, it is just a waste.